syne_tune/experiments/plotting.py

```diff
@@ -434,11 +435,28 @@ def _figure_shape(plot_params: PlotParameters) -> Tuple[int, int]:
             nrows = ncols = 1
         return nrows, ncols
 
+    def _transform_dataframe(
+        self,
+        df: pd.DataFrame,
+        plot_params: PlotParameters,
```

Comment:
We should just pass metric directly to this function.

Reply:
OK